xpk/xpk.py

```diff
@@ -257,6 +257,9 @@ class SystemCharacteristics:
   chips_per_vm: int
 
 ################### Subcommand Helper Functions #############
+""" !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+IF YOU MODIFY THE UserFacingNameToSystemCharacteristics MAP YOU SHOULD ALSO ADD CORRESPONDING MODICATIONS TO 
```

Comment:
Put an End for this warning! Do some ASCII art ^^^^^^^^

Reply:
Done.